plugins/VRPNPlugin/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2015-2018, EPFL/Blue Brain Project
+# All rights reserved. Do not distribute without permission.
+#
+# This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+
+project(VRPNPlugin VERSION 1.0.0)
+set(VRPNPlugin_VERSION_ABI 1)
+
+include(Common)
+
+#common_find_package(Brayns REQUIRED)
```

Comment:
have you tried to move adding the plugins after `include(CommonCPack)` to see if it works then?

Reply:
yes, does not work either